tests/e2e/smoke-restarting-deployment/00-errors.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: restarting-collector
+spec:
+  ports:
+  - targetPort: 4317
```

Comment:
Does this file define that the service should not expose 4317 port?

Reply:
yes, because we use it when we change the configuration and we have to check if it is not already.